Libraries/Hotcakes.PaypalWebServices/RestPaypalApi.cs

```diff
@@ -0,0 +1,288 @@
+            string phone, string invoiceId, bool isNonShipping)
+        {
+            HttpResponse response;
+            // Construct a request object and set desired parameters
+            // Here, OrdersCreateRequest() creates a POST request to /v2/checkout/orders
+            OrderRequest order = new OrderRequest()
+            {
+                CheckoutPaymentIntent = paymentAction,
+
+                ApplicationContext = new ApplicationContext
+                {
+                    CancelUrl = cancelURL,
+                    ReturnUrl = returnURL,
+                },
+                PurchaseUnits = new List<PurchaseUnitRequest>
+                    {
+                        new PurchaseUnitRequest
+                        {
+                            InvoiceId = invoiceId,
+                            AmountWithBreakdown = new AmountWithBreakdown
+                            {
+                                CurrencyCode = currencyCodeType,
+                                Value = orderTotal,
+                                AmountBreakdown = new AmountBreakdown
+                                {
+                                    ItemTotal = new Money
+                                    {
+                                        CurrencyCode = currencyCodeType,
+                                        Value = itemsTotal
+                                    },
+                                    Shipping = new Money
+                                    {
+                                        CurrencyCode = currencyCodeType,
+                                        Value = shippingTotal
+                                    },
+                                    TaxTotal = new Money
+                                    {
+                                        CurrencyCode = currencyCodeType,
+                                        Value = taxTotal
+                                    },
+                                }
+                            },
+                            ShippingDetail = new ShippingDetail
+                            {
+                                Name = new Name
+                                {
+                                    FullName = name
+                                },
+                                AddressPortable = new AddressPortable
+                                {
+                                    AddressLine1 = street1,
+                                    AddressLine2 = street2,
+                                    AdminArea2 = city,
+                                    AdminArea1 = region,
+                                    PostalCode = postalCode,
+                                    CountryCode = countryISOCode,
+                                }
+                            }
+                        }
+                    }
+            };
+
+            var request = new OrdersCreateRequest();
+            request.Prefer("return=representation");
```

Comment:
Please make static strings like this a `constant`.

Reply:
OK done it.